### PR TITLE
Update card.style.mjs

### PR DESCRIPTION
**Pull request by vsfarooqkhan**
Maximum width must be 60% which is compatible with all the browsers and mobile view. 
-> If You don't want to reduce the max-height and min-height , just close the pr  .
Thanks .

### CODE REVIEW

File: src/card.style.mjs [hunk 1]
```diff
@@ -71,8 +71,8 @@ export const css =
     }
 
     #content{
-        max-height:90%;
-        min-height:90%;
+        max-height:60%;
```
Comment (prafulla-codes) [inline src/card.style.mjs:74]: this just changes the height of the content and not of the card

**Comment by prafulla-codes** [comment]
Hi thanks, I agree that the width should be 60% but the code you have changed changes the height of the content (check netlify preview of the changes you have made)

**Comment by vsfarooqkhan** [comment]
That's what Actually I meant for. I have tried to implement it on my personal website. But it overwrites the existing content with it's height of '90%'.  
Capture

**Comment by prafulla-codes** [comment]
Ok I'll check it out, and let you know why it's getting over written

**Comment by prafulla-codes** [comment]
The code that is over writing the width and height is in card.component.mjs

**Comment by vsfarooqkhan** [comment]
Okay. What is the solution ?

**Comment by prafulla-codes** [comment]
I'll change the file and push it, i'll let you know once it's done. Please let me know if that is the output you wanted

**Comment by vsfarooqkhan** [comment]
.

**Pull request closed without merging**